app/assets/stylesheets/custom.scss

```diff
@@ -20,3 +20,52 @@ abbr[title="required"] {
 .panel .panel-footer {
   margin-bottom: 0;
 }
+
+/* Devise flash */
+
+.alert-error {
```

Comment:
Is all of this styling because you wanted different `flash` messaging behavior than the standard bootstrap behavior?